### ISSUE TITLE
Measure and track code coverage for appsody cli tests

### ISSUE THREAD

**Issue by neeraj-laad** [appsody/appsody#55]
We would like to measure how much of hte appsody cli code are we testing and then gradually increase that coverage as we evolve the cli itself.

Possible options:
- https://golang.org/cmd/cover/

**Comment by kylegc** [comment]
Yes, I agree we should track test code coverage. Moving this under epic #44 

I looked into this in the past. `go test -cover` works well with our existing test tools. However, because of issue #36 the code exists before the test framework can process the results. Therefore we currently have tests calling appsody in forked processes which breaks code coverage. 

In short, #36 needs to be fixed before we can do this.
